archunit/src/test/java/com/tngtech/archunit/core/importer/ClassFileImporterAnnotationsTest.java

```diff
@@ -97,6 +97,21 @@ public void imports_annotation_defaults() {
                 .as("default of clazzWithDefault()").matchExactly(Serializable.class, List.class);
     }
 
+    @Test
+    public void distinguishes_between_explicitly_set_values_and_default_values() {
+        @SimpleAnnotationWithDefaultValue(hasDefaultButOverwritten = "hasDefaultButOverwritten", hasNoDefault = "hasNoDefault")
```

Comment:
I'd replace `"hasDefaultButOverwritten"` with `"overwritten"`, and `"hasNoDefault"` with `"value"`.

In addition, I'd love to test one more property with default, for which an explcit value is given which however coincides with the default.

Reply:
Fair point :smiley:

Reply:
I kind of like `hasNoDefault` though, because it makes it clearer what the point is here? :thinking:

Reply:
I mean, for "overwritten" it already implies there is a default :wink: But "value" does not really communicate that there is no default to me :thinking:

Reply:
As mentioned below, I have tried to put the specifics closer to the test. Do you think it's okay now?